bin/development-server.js

```diff
@@ -58,22 +58,17 @@ app.get("/", function(req, res) {
   res.sendFile(path.join(__dirname, "../index.html"));
 });
 
-app.get("/chrome-tabs", function(req, res) {
-  if(feature.isEnabled("chrome.debug")) {
-    const webSocketPort = feature.getValue("chrome.webSocketPort");
-    const url = `http://localhost:${webSocketPort}/json/list`;
-
-    const tabRequest = httpGet(url, body => res.json(JSON.parse(body)));
-
-    tabRequest.on('error', function (err) {
-      if (err.code == "ECONNREFUSED") {
-        console.log("Failed to connect to chrome");
-      }
-    });
-  }
-  else {
-    res.json([]);
-  }
+app.get("/get", function(req, res) {
```

Comment:
converting this custom endpoint to a generic endpoint that handles cross origin http requests